content/en/docs/kitex/Tutorials/advanced-feature/metainfo.md

```diff
@@ -88,6 +88,14 @@ func main() {
         val, ok := metainfo.RecvBackwardValue(ctx, "something-from-server") // receive the meta information from server side
         println(val, ok)
     }
+
+    // receive all the meta information from server side
+		m := metainfo.RecvAllBackwardValues(ctx) // m: map[string]string
```

Comment:
indent not match with the rest. same below.

Reply:
all fixed and checked.